e2e/playwright/sketch-tests.spec.ts

```diff
@@ -1352,7 +1352,7 @@ test.describe(`Sketching with offset planes`, () => {
   })
 })
 
-test.describe('multi-profile sketching', () => {
+test.describe('multi-profile sketching', { tag: '@skipWin' }, () => {
```

Comment:
I've seen flakes for a few of these on Windows. We have coverage on Ubuntu and macOS, Windows isn't adding anything here.